### PR TITLE
Project popup

### PR DESCRIPTION
**Pull request by KanzaTahreem**
This pull request covers the following requirements

### Project requirements

- You should implement the following interactions:
  - When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
  - When the user clicks (or taps) the close (X) button, the popup disappears.

- In order to associate each project with the popup window details, you must refactor the project section:
  - You need to use **a JavaScript array to store all of the information for all projects**.
  - For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, and link to the source.
  - You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.
  - After that, you could implement the popup window.
- You should implement popups for both mobile and desktop screen sizes.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -15,4 +15,137 @@ window.onload = () => {
...
     popUp.classList.add('hidden');
   });
-};
\ No newline at end of file
+
+  const projectsData = [
+    {
+      title: 'Tonic',
+      subtitle: 'CANOPY',
+      techStack: 'Back End Dev',
```
Comment (tufoinnkuo10) [inline script.js:23]: - [ ] great job so far. please ensure that the mobile version of popup is full heigth like that of figma

yours
Screenshot (429)_LI
figma
Screenshot (430)

File: script.js [hunk 2]
```diff
@@ -15,4 +15,137 @@ window.onload = () => {
...
+      year: 2015,
+      src: './images/second-section/project-01.svg',
+      highlight: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
```
Comment (tufoinnkuo10) [inline script.js:26]: - [ ] please kindly add the popup image size for the desktop version so it looks like that of figma
- [ ] increase the close icon a little and bring it inside to make it more visible
- [ ] Kindly add your description font size(16px) like that of figma

yours
Screenshot (435)_LI
figma
Screenshot (431)

File: script.js [hunk 3]
```diff
@@ -15,4 +15,137 @@ window.onload = () => {
...
+      lang: ['html', 'css', 'javascript'],
+      live: 'https://kanzatahreem.github.io/Portfolio-Website/',
+      source: 'https://github.com/KanzaTahreem/Portfolio-Website',
+    },
+    {
+      title: 'Multi Post Stories',
+      subtitle: 'CANOPY',
+      techStack: 'Back End Dev',
+      year: 2015,
+      src: './images/second-section/project-02.svg',
```
Comment (tufoinnkuo10) [inline script.js:37]: - [ ] please kindly make the mobile version responsive when switching to deskstop version
Screenshot (435)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @KanzaTahreem 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ popup goten dynamically from script array
✅ design well followed

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by dasileker — APPROVED** [review]
### **Hi @KanzaTahreem,** 

### **Congratulations! 🎉**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 : 
 
### **Note**

- `Please make sure that your readme file is having the right Author name right you have multiple Authors.`


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**